CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Pin docker dependency to fix docker-compose incompatibility in
+  docker proto tests
+  ([#3611](https://github.com/open-telemetry/opentelemetry-python/pull/3611))
+
```

Comment:
There is no need for CHANGELOG entry as this is not useful for end users

Reply:
I agree, I added it because the changelog job fails. Removed it now.